virtualfish/environment.fish

```diff
@@ -1,6 +1,30 @@
+# Set up the configurable filename
+if not set -q VIRTUALFISH_ENVIRONMENT_FILE
+    set -g VIRTUALFISH_ENVIRONMENT_FILE .env
+end
+
+function __vfsupport_set_env_file_path --description "Set VIRTUALFISH_ENVIRONMENT_FILE_PATH to appropriate value"
+    set -l vf_base (basename $VIRTUAL_ENV)
+    set -l vf_path $VIRTUAL_ENV/$VIRTUALFISH_ENVIRONMENT_FILE
+    # Check if projects plugin is used
+    if set -q PROJECT_HOME
+        set -l project_path $PROJECT_HOME/$vf_base/$VIRTUALFISH_ENVIRONMENT_FILE
+        if not test -r $project_path
+            and test -r $vf_path
+            set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $vf_path
+        else
+            set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $project_path
+        end
+    else
+        set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $vf_path
+    end
+end
```

Comment:
It wasn't until I was working on the environment command that I realized that was an issue, just because it opens the file in the `$EDITOR` regardless of whether it exists. Kind of an edge case, but it caught me. 😄